packages/android-app/android/build.gradle

```diff
@@ -0,0 +1,68 @@
+import groovy.json.JsonSlurper
+
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    
+    repositories {
+        google()
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.1'
+        
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
+
+def reactRepositoryPath = "$rootDir/../assets/react-native-modules"
+
+task removePreviousReactArchives(type: Delete) {
+    delete reactRepositoryPath
+}
+
+task generateReactArchives {
+    subprojects { subproject ->
+        apply plugin: 'maven'
+        def reactProjects = ['react-native-linear-gradient', 'react-native-svg']
```

Comment:
should this include also `react-native-device-info`?

Reply:
Our current native app doesn't have device-info and it seems to work without the native package. Might be a good idea to check how it works without it

Reply:
Date publications have a fallback when it can't find device-info package and defaults to gmt. Now I added device info to android project and xnative bundling so it picks the correct timezone.